GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
+                        x = a[i]
+                        y = x[0]
+                        # print(y)
+                        names.append(y)
+                    # print(names)
+
+                    # Local Administrators are added to list of names
+                    EXEC = sub.Popen(["powershell", "& {net localgroup administrators}"],
+                                     stdout = sub.PIPE)
+                    stdout, _ = EXEC.communicate()
+                    output = stdout.decode("utf-8")
+                    output = output.split("\n")
+                    # print(output)
+                    n2 = []
+                    for i in range(6, len(output) - 3):
+                        # print(output[i])
+                        # if '\r' in output[i]:
+                        #     print('found in ' + str(i))
+                        #     print(output[i])
+                        n2.append(output[i])
+                    a2 = []
+                    for i in range(0, len(n2)):
+                        x = n2[i]
+                        y = x.split('\r')
+                        # print(y)
+                        a2.append(y)
+                    # print(a)
+                    for i in range(0, len(a2)):
+                        # print(a[i])
+                        x = a2[i]
+                        y = x[0]
+                        EXEC = sub.Popen(["powershell", "& {$env:UserName}"], stdout = sub.PIPE)
+                        stdout, _ = EXEC.communicate()
+                        output = stdout.decode("utf-8")
+                        output = output.split('\r\n')
+                        if output[0] == y:
+                            y = output[0] + '   (Current User)'
+                        # print(y)
+                        names.append(y)
+                    # print(names)
+                    return names
+
+                listoNames = findnames()
+
+                for i in range(0, len(listoNames)):
+                    QListWidgetItem(listoNames[i], self.listOFnames)
+
+                def removal():
+                    username = self.Username_Input.text()
+
+                    def completedPOP(username):
+                        COMPLETE = QMessageBox()
+                        COMPLETE.setIcon(QMessageBox.Question)
+                        COMPLETE.setWindowTitle('Hey! Listen!')
+                        COMPLETE.setText(
+                            'User ' + username + ' has been successfully removed from the system.')
+                        COMPLETE.setStandardButtons(QMessageBox.Close)
+                        COMPLETE.exec_()
+                        self.close()
+
+                    if len(username) == 0 or username not in listoNames:
+                        if len(username) == 0:
+                            print('no username entered')
+                            ERROR_NO_USER = QMessageBox()
+                            ERROR_NO_USER.setIcon(QMessageBox.Warning)
```

Comment:
Codacy found an issue: ['QMessageBox' may be undefined, or defined from star imports: PyQt5.QtWidgets, threading (F405)](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927260)